website/docs/language/meta-arguments/enabled.mdx

```diff
@@ -0,0 +1,86 @@
+---
+description: >-
+  The enabled meta-argument allows you to conditionally create or skip a single
+  resource or module instance based on a boolean value.
+---
+
+# The `enabled` Meta-Argument
+
+:::note
+A given resource or module block cannot use `enabled` together with `count` or `for_each`.
+:::
+
+The `enabled` meta-argument in a `lifecycle` block provides a cleaner way to conditionally create or skip a
+single resource or module instance. Unlike the traditional `count = var.enabled ? 1 : 0`
+workaround, which requires array indexing to access single instances, `enabled` allows
+direct access when the resource is created and properly handles the null state when disabled.
+
+## Basic Syntax
+
+`enabled` is a meta-argument that goes inside a `lifecycle` block. It accepts expressions that turn into boolean values to determine whether the resource or module should be created.
+
+```hcl
+variable "create_instance" {
+  type    = bool
+  default = true
+}
+
+resource "aws_instance" "example" {
+  # ...
+  lifecycle {
+    enabled = var.create_instance
+  }
+}
+```
+
+When `enabled` is `true` (or not specified), the resource behaves normally. When `enabled`
+is `false`, the resource is not created and evaluates to `null`.
+
+## Accessing Disabled Resources
+
+When a resource might be disabled, it evaluates to `null` and cannot be accessed directly.
+You must use conditional expressions to handle this safely:
+
+```hcl
+# Using try() - most concise but masks other errors
+output "instance_id" {
+  value = try(aws_instance.example.id, "not-created")
+}
+
+# Using can() - more explicit about null checking
+output "instance_id" {
+  value = can(aws_instance.example.id) ? aws_instance.example.id : "not-created"
+}
```

Comment:
This example is essentially equivalent to the `try` example above... `try` is effectively just a shorthand for a conditional based on `can`, like this one.

I suggest we remove this and focus only on the `try` and `!= null` examples.